types/validator.yaml

```diff
@@ -35,20 +35,34 @@ Validator:
 AttesterDuty:
   type: object
   properties:
-    validator_pubkey:
+    public_key:
```

Comment:
This is called `pubkey` everywhere in the spec, I would consider following the convention here